app/src/main/java/com/android/solvit/seeker/ui/profile/SeekerRegistrationProfile.kt

```diff
@@ -236,17 +233,23 @@ fun SeekerRegistrationScreen(
                           painter = painterResource(id = R.drawable.userpref),
                           contentDescription = "Completion Image",
                           modifier =
-                              Modifier.size(300.dp)
+                              Modifier.size(100.dp)
                                   .align(Alignment.CenterHorizontally)
                                   .testTag("preferencesIllustration"))
                       Spacer(modifier = Modifier.height(20.dp))
-                      Text(
-                          text = "This feature is not implemented yet.",
-                          style = MaterialTheme.typography.bodyLarge,
-                          modifier = Modifier.align(Alignment.CenterHorizontally),
-                          textAlign = TextAlign.Center,
-                          color = colorScheme.primary)
-                      Spacer(modifier = Modifier.height(100.dp))
+                      val userId = user?.uid
+                      if (userId != null) {
+                        Preferences(userId, viewModel)
+                      } else {
+                        Text(
```

Comment:
I think that this edge case has no to be implemented here, if the user is not authenticated, he doesn't even have to switch on this screen

Reply:
Thank you for your feedback. I wanted to clarify that after logging out following registration, the user is redirected to the registration screen. Since an unlogged user cannot access the preferences section, I highlighted the need for the user to be logged in.